Builds/CMake/deps/Lz4.cmake

```diff
@@ -75,5 +77,7 @@ endif()
 
 add_dependencies (lz4_lib lz4)
```

Comment:
I know it's part of the original script. But shouldn't we have this dependency only when lz4 is added via ExternalProject_Add? lz4 is a custom target created by ExternalProject_Add.

Reply:
My suspicion is that it's going to be needed by all builds because `lz4` can also be discovered by `find_package` on line 8.

There's an easy way to find out, though instead of just speculating: https://github.com/ripple/rippled/pull/3532/commits/4773981698ed3a754f1b5496a9710eda730956fa
And I'll check back on the Travis build later to see if it succeeds. If it does, then the dependency isn't necessary when found. If it doesn't then I'll rollback that commit. https://travis-ci.com/github/ripple/rippled/builds/176060773

Reply:
Sounds good

Reply:
Build worked, so I'll keep that commit.